### ISSUE TITLE
Jump to position in 3D viewport

### ISSUE THREAD

**Issue by georgwiese** [scalableminds/webknossos#5508]
## Detailed Description
When pasting positions, all 3 viewports jump to the new position, but the 3D viewport always has to be moved manually. It would be great if it would either behave the same or perhaps have a button to jump to the current position.

## Context
I'm proofreading "endings" found in skeletons by our automatic ending detection.

**Comment by philippotto** [comment]
I agree, that this is probably useful most of the time :+1: 

> It would be great if it would either behave the same or perhaps have a button to jump to the current position.

The existing 3D/XY/... buttons don't do the trick for you because they change rotation/zoom?

**Comment by georgwiese** [comment]
Yes, exactly, they would just show me the whole dataset, but I'd like to look at different spots in a certain zoom level (I guess I adjust the rotation anyway for each location).
